pyproject.toml

```diff
@@ -1,50 +1,3 @@
 [build-system]
-requires = ['poetry>=0.12']
-build-backend = 'poetry.masonry.api'
-
-[tool.poetry]
-name = 'build'
-version = '0.0.3.1'
-description = 'A simple, correct PEP517 package builder'
-license = 'MIT'
-
-authors = ['Filipe Laíns <lains@archlinux.org>']
-maintainers = [
-    'Filipe Laíns <lains@archlinux.org>'
-]
-
-homepage = 'https://github.com/FFY00/python-build'
-repository = 'https://github.com/FFY00/python-build'
-documentation = 'https://python-build.readthedocs.io/en/stable/'
-readme = 'docs/source/index.rst'
-
-classifiers = [
-    'License :: OSI Approved :: MIT License',
-    'Programming Language :: Python :: 2.7',
-    'Programming Language :: Python :: 3',
-]
-
-packages = [
-    { include = 'build' },
-]
-
-
-[tool.poetry.dependencies]
-# https://github.com/python-poetry/poetry/issues/2542
-#python = [
-#    '2.7',
-#    '>=3.5',
-#]
-toml = '^0.10'
-pep517 = '^0.8'
-packaging = '^20.0'
-importlib-metadata = { version = '^1.0', markers = 'python_version<"3.8"' }
-typing = { version = '^3.0', markers = 'python_version<"3"' }
-
-[tool.poetry.dev-dependencies]
-pytest = '^5.0'
-pytest-mock = '^3.0'
-pytest-cov = '^2.0'
-
-[tool.poetry.scripts]
-python-build = 'build.__main__:main'
+requires = ['setuptools >= 40.8.0', 'wheel']
+build-backend = 'setuptools.build_meta:__legacy__'
```

Comment:
Why the legacy backend?

Reply:
Yeah definitely don't use this. It's intended for backwards compatibility only.

Reply:
Ah, sorry. I just defaulted to `__legacy__` because it's our fallback on missing `build-backend`.